.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,68 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
```

Comment:
Change these to:

```yaml
on:
  workflow_dispatch:
```

and this workflow will only be activated by manual triggering from the Actions menu

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch